HtmlMesh/src/html-mesh.ts

```diff
@@ -6,6 +6,114 @@ import { PointerEventsCaptureBehavior } from "./pointer-events-capture-behavior"
 import { Scene } from "@babylonjs/core";
 import { Logger } from "@babylonjs/core/Misc/logger";
 
+export enum SizeFit {
+    none = 1,
+    contain = 2,
+    cover = 3,
+    stretch = 4
+}
+
+export type SizeFitHandler = {
+    handleElement(element: HTMLElement): HTMLElement
+    handleSizeFit(parent: HTMLElement, width: number, height: number): void
```

Comment:
The implementations never seem to use parent element so why don't we just make the parameter the sizing element (the element returned by wrapElement).